platform/jobs/edxPlatformPrTestsSpecialWorker.groovy

```diff
@@ -0,0 +1,48 @@
+package platform
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+buildFlowJob('edx-platform-pr-tests-special-worker') {
```

Comment:
Not a must, but could we use the build descriptor plugin <sic> to assign a description to the job so it's easy to look at past jobs and understand what they were run for?